src/index.d.ts

```diff
@@ -270,10 +270,13 @@ export type VariantTypeSchema = {
     name: string
     id: string
     tag: number
-    fieldTypes: FieldTypeSchema[]
+    fieldTypes: FieldTypeSchema<"plain">[]
```

Comment:
Instead of making this generic, I would create a separate `Cip68StructTypeSchema` with `format: "map"`, and then create a separate `Cip68FieldTypeSchema` with the `key: string` field. Also recommend keeping the field names short because these typeschemas are inlined everywhere in the bundle produced by the contract-utils

Reply:
okay, I think that works for me.  Thanks

Reply:
done